jobs/models.py

```diff
@@ -89,13 +89,18 @@ class Meetup(models.Model):
     description = models.TextField()
     is_recurring = models.BooleanField(
         default=False,
-        help_text="Is your meetup recurring?"
     )
     #TODO this field should be required if the is_recurring is True
-    recurrence = models.CharField(max_length=255, blank=True, null=True)
+    recurrence = models.CharField(
+        max_length=255,
+        blank=True,
+        null=True,
+        help_text="Provide reccurence if your meetup is reccuring."
```

Comment:
typo "recurrence";  would record to "Provide details of recurrence if applicable"